RUFAS/routines/field/soil/soil_data.py

```diff
@@ -258,19 +250,18 @@ class SoilData:
     eroded_active_organic_nitrogen: float = 0.0
 
     # ---- Residue partition (Carbon Cycling)
-    plant_surface_residue = 0.0
-    plant_root_residue = 0
     plant_residue_lignin_composition: float = 0.17
     plant_lignin_nitrogen_ratio: float = 0
     plant_residue_metabolic_fraction: float = 0
-    total_residue: float = 0
-    crop_root_depth: float = 0
     crop_yield_nitrogen: float = 0
 
     @property
-    def all_residue(self) -> float:  # TODO: not currently used.
-        """amount of total plant residue, above and below-ground, on the field (kg/ha)"""
-        return self.plant_surface_residue + self.plant_root_residue
+    def all_residue(self) -> float:
```

Comment:
```suggestion
    def total_residue(self) -> float:
```
The word "all" can be confused with the Python keyword `all()`.

Reply:
Good call!